docs/landing-page.md

```diff
@@ -0,0 +1,110 @@
+# Landing page
+
+_This feature is an experimental feature. The setup and configuration of the landing page needs manual code editing. There's no user interface in Admin panel to configure landing page._
+
+This guide will help you to setup and configure landing page. When the landing page is enabled, it will be served to your users from the root URL of your marketplace. The old root search view will be found from `/s` path.
+
+## Enabling landing page
+
+To enable the landing page, change the `clp_static_enabled` to `true`
+
+```yaml
+# config.yml
+
+clp_static_enabled: true
+```
+
+Restart the server and go to the marketplace root URL. You should now see the landing page with the example content.
+
+## Advanced configuration
+
+### Caching
+
+The landing page is cache heavily in order to make it lightning fast. There are two configurations that control caching:
+
+- `clp_cache_time`: The time in seconds how often the cache is invalidated. Default: 900 seconds, i.e. 15 minutes
+- `clp_static_released_version`: The current landing page version. The version number acts as a cache buster. The cache is in any case invalidated after `clp_cache_time`, but if you want to invalidate the cache sooner, increase the version number.
```

Comment:
Mention that the landing page is also cached browser-side (we send `Cache-Control: max-age=#{clp_cache_time}`), so some users would still see older version, even if the app-side cache is invalidated.